app/src/routes/auth/authRoutes.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+import express from "express";
+import ctrl from "./authCtrl.js"
+import jwt from "jsonwebtoken";
+
+const router = express.Router();
+
+router.post("/login", (req, res, next) => {
+  let token = req.headers['authorization'];
+  if (!token) {
+    const response = { error: 'Bad Request', message: "토큰이 없습니다.", statusCode: 400 }
+    return res.status(response.statusCode).json(response);
+  }
+  const [type, clientAccessToken] = token.split(' ');
+  try {
+    let payload = jwt.verify(clientAccessToken, process.env.ACCESS_TOKEN_SECRET);
```

Comment:
```suggestion
    const payload = jwt.verify(clientAccessToken, process.env.ACCESS_TOKEN_SECRET);
```
let으로 선언할 이유가 없어 보이는데요